operations/helm/charts/mimir-distributed/templates/memcached/_memcached-statefulset.tpl

```diff
@@ -131,6 +131,26 @@ spec:
             {{- toYaml . | nindent 12 }}
             {{- end }}
 
+          {{- if $.ctx.Values.memcached.readinessProbe.enabled }}
+          readinessProbe:
+            tcpSocket:
+              port: {{ .port }}
+            initialDelaySeconds: {{ $.ctx.Values.memcached.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ $.ctx.Values.memcached.readinessProbe.periodSeconds }}
+            successThreshold: {{ $.ctx.Values.memcached.readinessProbe.successThreshold }}
+            failureThreshold: {{ $.ctx.Values.memcached.readinessProbe.failureThreshold }}
+          {{- end }}
+      
+          {{- if $.ctx.Values.memcached.livenessProbe.enabled }}
+          livenessProbe:
```

Comment:
A liveness probe doesn't make sense for Memcached. A liveness probe means that a pod is broken and cannot recover except by being restarted. This isn't the case for Memcached, it should always be able to recover as long as it's still running.